src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -5489,7 +5490,63 @@ public void testListNodes() throws Exception {
       assertThat(plugin.name(), allOf(notNullValue(), not("")));
     }
   }
-  
+
+  @Test
+  public void testInspectNode() throws Exception {
```

Comment:
Just updated this test class and removed this test method in favor of the unit tests.